### DISCUSSION TITLE
Jio Tv not playing again from KODI

### DISCUSSION THREAD

**Discussion by masternkv** [community/community#45683] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Jio tv not playing

**Comment by queenofcorgis** [comment]
Hi @masternkv ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on https://www.jiotv.com/. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
